cscore/src/main/native/windows/UsbCameraImpl.h

```diff
@@ -99,7 +99,7 @@ class UsbCameraImpl : public SourceImpl,
         : kind(kind_), data{0}, from(std::this_thread::get_id()) {}
 
     Kind kind;
-    int data[4];
+    int data[4]{0};
```

Comment:
This looks like a false positive?  The constructor initializes it.

Reply:
It's not. It's recommending member init instead of ctor init list.
```
/home/tav/frc/wpilib/allwpilib/cscore/src/main/native/windows/UsbCameraImpl.h:102:9: warning: use default member initializer for 'data' [modernize-use-default-member-init]
    int data[4];
        ^
               {0}
```